webpack/ForemanTasks/Components/TasksTable/__tests__/__snapshots__/TasksTableActions.test.js.snap

```diff
@@ -138,6 +138,7 @@ Array [
   ],
   Array [
     [Function],
+    undefined,
```

Comment:
should it be `undefined` ?

Reply:
Yes, its from the `
  dispatch(fetchTasksSummary(getURIQuery(url).time), parentTaskID);` line
as `resumeTask` didn't sent a value to `parentTaskID`